src/main/java/racing/InputUtil.java

```diff
@@ -0,0 +1,14 @@
+package racing;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class InputUtil {
+
+    public String inputNumber(){
+        Scanner scanner = new Scanner(System.in);
```

Comment:
다음과 같이 Scanner를 한번만 생성하도록하면 어떤 장점이 있을까요 :) ?
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
    
    public String inputNumber() {
        
```